### DISCUSSION TITLE
Reminders not working

### DISCUSSION THREAD

**Discussion by vravikumar98** [community/community#133677] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I have a reminder setup with slack. I was receiving reminders yesterday (7/26), but am not seeing any today. I tried sending a test reminder from https://github.com/settings/reminders and didn't receive that either.

**Comment by setu4993** [comment]
Running into the same issue, reported it on the Slack integration repo here: https://github.com/integrations/slack/issues/1927

**Comment by glasser** [comment]
Seeing this as well.

**Reply by glasser** [reply]
I had filed a support ticket too and I just heard back that it has been escalated to engineering.

**Comment by artemlive** [comment]
Same thing for me.

**Comment by somersby10ml** [comment]
same

**Comment by sfreydin** [comment]
Same for us

**Comment by sebastian-fredriksson-bernholtz** [comment]
Same for everyone I've checked with at our company as well.

**Comment by francisc-neculau** [comment]
same

**Comment by e-krebs** [comment]
same issue is happening since Friday
